xml/security_pcidss.xml

```diff
@@ -7,272 +7,1251 @@
+     way to keep systems up-to-date. It offers seamless management of both
+     &productname; and &rhel; client systems. This
+     is particularly handy in larger system environments, to check the
+     current update status of each system and to react on know security risks.
+    </para>
+    <para>
+     Most parts of this requirement are about in-house security software
+     development, documentation and design questions. In this prospect we
+     will keep our scope on how to keep the operating system maintained and
+     secure.
+    </para>
+    <variablelist>
+     <varlistentry>
+      <term>
+       6.2.a Examine policies and procedures related to security patch
+       installation to verify processes are defined for: (1) Installation of
+       applicable critical vendor-supplied security patches within one month
+       of release, (2) Installation of all applicable vendor-supplied
+       security patches within an appropriate time frame (for example, within
+       three months).
+      </term>
+      <listitem>
+       <para>
+        To identify patches that
+        need to be installed to secure your system, do the following:
+       </para>
+       <!-- FIXME: procedure -->
+       <para>
+        First refresh all software repositories so that we have up-to-date information.
+       </para>
+       <screen>zypper refresh</screen>
+       <para>
+        Now we can look for important security fixes that have not yet been
+        installed:
+       </para>
+       <screen>zypper list-patches --category security --severity important</screen>
+       <para>
+        It is also possible to search for CVE or SUSE Bugzilla numbers. By
+        default only needed patches are listed by this command. To show also
+        patches that have already been installed use <option>--all</option>:
+       </para>
+       <screen>zypper list-patches --all --cve=CVE-2016-4957</screen>
+       <para>
+        Details of individual patches can be listed with the patch-info
+        subcommand:
+       </para>
+       <screen>zypper patch-info SUSE-SLE-SERVER-12-SP1-2016-600</screen>
+       <para>
+        To install only important security patches use the patch command. All
+        <command>zypper</command> commands can be run with the <option>--non-interactive</option> to run
+        automatically<!--fixme: wording-->:
+       </para>
+       <screen>zypper patch --category security --severity important</screen>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+   </sect3>
+  </sect2>
+  <sect2>
+   <title>Implement Strong Access Control Measures</title>
+   <para/>
+   <sect3>
+    <title>Requirement 7: Restrict access to cardholder data by business need to know</title>
+    <para>Summary:</para>
+    <para>
+     Access control is a complex topic if it comes to the operating system.
+     Again, the &pcidssa; requirement is quite fuzzy at this point and does not
```

Comment:
<del>is quite fuzzy</del> is not currently specified precisely <del>at this point</del>